aui.core/src/AUI/Logging/ALogger.h

```diff
@@ -171,13 +171,15 @@ class API_AUI_CORE ALogger final
                 mLogger(logger),
                 mLevel(level),
                 mPrefix(std::move(prefix)) {
-
             }
 
             ~LogWriter() {
                 mBuffer.write(0); // null terminator
                 auto s = mBuffer.str();
                 mLogger.log(mLevel, mPrefix.toStdString().c_str(), s);
+                if (mLogger.mOnLogged) {
+                    mLogger.mOnLogged(mPrefix, s, mLevel);
+                }
```

Comment:
Why not put this call to ALogger::log function?

Reply:
for real